docs/translating/spanish.md

```diff
@@ -34,10 +34,10 @@ Zulip friendly and usable.
 ## Términos
 
 - Message - **Mensaje**
-- Private message (PM) - **Mensaje privado (MP)**
-- Group PM - **mensaje privado grupal**: even though "MP grupal" is the most
+- Direct message (DM) - **Mensaje directo (MD)**
+- Group PM - **mensaje directo grupal**: even though "MD grupal" is the most
   precise translation, preferably don't use that one. Many users may not
-  associate "MP" with _private message_ in a group context. Better use it
+  associate "MD" with _direct message_ in a group context. Better use it
```

Comment:
I feel like this "Better use it without abbreviations" is a bit of an odd translation of "Mejor usarlo sin abreviaturas", which really should be "Better to use it without the abbreviation" or "It's better not to use the abbreviated form."

I guess as a whole I might revise this to be:
> Many users may not associate "MD" with _direct message_ in a group context, so it's better to use the unabbreviated form.

Thoughts?

I'm also not a fan of "preferably don't use that one".

Reply:
Yes, good point! Hmm, but maybe we should make it a bit more explicit:

> Many users may not associate "MD" with _direct message_ in a group context, so it's better to use the unabbreviated form rather than "MD grupal".

Reply:
That works for me.